packages/ts/frontend/src/Connect.ts

```diff
@@ -24,6 +24,8 @@ $wnd.Vaadin.registrations.push({
   is: 'endpoint',
 });
 
+export const bodyPartName = 'hilla_body_part';
```

Comment:
I know, we are not good in following this rule but let's try to keep exported const in the uppercase.
```suggestion
export const BODY_PART_NAME = 'hilla_body_part';
```